contact-api/spec/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>Redirecting to https://w3c.github.io/contact-picker/spec/</title>
+<meta content="0; URL=https://w3c.github.io/contact-picker/spec/"
+      http-equiv="refresh" />
```

Comment:
collecting data 📊 is the key 🔑 in making the best educated and most efficient way you are to get your business 👩‍💼 going right now is one ☝🏼 
<img width="148" alt="IMG_2102" src="https://github.com/user-attachments/assets/d1306d40-5703-4814-87d5-f37bfced7684">